proxyd/backend.go

```diff
@@ -692,6 +700,11 @@ func (bg *BackendGroup) Forward(ctx context.Context, rpcReqs []*RPCReq, isBatch
 				})
 				if errors.Is(err, ErrRewriteBlockOutOfRange) {
 					res.Error = ErrBlockOutOfRange
+				} else if errors.Is(err, ErrRewriteRangeTooLarge) {
+					res.Error = &RPCErr{
+						Code:    JSONRPCErrorInternal,
```

Comment:
[Nit] Should be an invalid params error - can use `ErrInvalidParams()` to generate this.